acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/ethsigner/testutil/EthSignerConfig.java

```diff
@@ -66,6 +75,11 @@ public Integer getDownstreamHttpPort() {
     return downStreamHttpPort;
   }
 
+  @Override
+  public String getDownstreamHttpPath() {
+    return "/";
```

Comment:
Is there a reason for not returning the path?

Reply:
This path enables serving json-rpc calls other than root ... for example `http://localhost:8888/custom-path` ... Besu AT starts and expects ethsigner to be serve json-rpc on root path i.e. '/'. If you start ethsigner directly, this path is default.